.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+* **Internal Jira link(s):**
```

Comment:
We cannot add internal links to open source.

Reply:
I understand the benefits and it's your decision, just be aware that it violates the ope-source policy.

Reply:
removed the jira link